lib/color-formatters.js

```diff
@@ -7,6 +7,9 @@
 const moment = require('moment');
 
 function version(version) {
+  if ((version === undefined) || (version === null)) {
```

Comment:
would we be better to check `typeof(version) !== 'string' && typeof(version) !== 'number'`?

Reply:
Good call. I've updated to guard against other spurious inputs